Mozillians_Repositioning.md

```diff
@@ -0,0 +1,233 @@
+# Mozillians.org Repositioning Architecture
+
+* [Mozillians.org Services](#mozilliansorg-services)
+  + [Technologies (Common)](#technologies-common)
+  + [New Front-End](#new-front-end)
+  + [Mozillians.org](#mozilliansorg)
+  + [Search Service](#search-service)
+  + [Profile Publisher](#profile-publisher)
+  + [Orgchart Service](#orgchart-service)
+* [Integrity Checker](#integrity-checker)
+* [CIS requirements](#cis-requirements)
+  + [Profile Updates from Mozillians](#profile-updates-from-mozillians)
+  + [Profile Updates from external Sources](#profile-updates-from-external-sources)
+
+
+## Mozillians.org Services
+
+### Technologies (Common)
+
+#### GraphQL
+
+We stated a discussion around whether we want to test [GraphQL](https://graphql.org/) as our API
+query language. Could be a great fit and we plan to give it a shot.
```

Comment:
we had a quick chat with @akatsoulas - we though of also using GraphQL for person-api with similar libraries.
If we do do that, it'd be interesting to have the GraphQL implementation from mozillians.org to be as close as possible so that you could switch over to person-api over time.
Most likely something of the sort:
```
mozillians.org => graphql-cache => graphql => person-api
+ mozillians.org => graphql-cache => graphql => mozillians.org db
```
vs
```
mozillians.org => graphql => mozillians.org db
```
is this what you're thinking as well, or something different?

Reply:
We haven't designed for this yet, but we should keep this in mind.

I think our main use case is
```
front-end => graphql => mozillians.org
```
but it probably makes sense to use it for back-end communication, too.